README.md

```diff
@@ -13,7 +13,17 @@ Test common in-app issues
 - ✅ Test in-app downloads (on-the-fly and static assets)
 - ✅ Test in-app escape links (Android + iOS)
 
-# Mobile App Tests
+## Running
+
+- `nvm use`
```

Comment:
```suggestion
- `nvm install`
```

nit: can just use `nvm install` to install-and-use or use-if-already-installed. I have a quickie blog post about this I need to publish sometime soon.

Reply:
good to know! I'll put `nvm install` / `nvm use` for us old-timers 🤣